src/StreamingDataFrames/streamingdataframes/app.py

```diff
@@ -270,6 +288,20 @@ def stop(self):
         """
         self._running = False
 
+    def _quix_runtime_init(self, dataframe: StreamingDataFrame):
+        """
+        Do some runtime setup only applicable to an Application.Quix instance
+
+        :param dataframe: instance of `StreamingDataFrame`
+        """
+        if self._quix_config_builder:
```

Comment:
I think it's a good point to introduce some  `is_quix_app()` property vs checking the `self._quix_config_builder` everywhere

Reply:
done!